gnomad_qc/v4/resources/variant_qc.py

```diff
@@ -296,20 +296,24 @@ def get_variant_qc_result(
 
 
 def final_filter(
-    data_type: str = "exomes", test: bool = False
+    data_type: str = "exomes", test: bool = False, simplified: Optional[bool] = False
```

Comment:
Changing to make it a little more obvious than simplified 
```suggestion
    data_type: str = "exomes",
    all_variants: bool = False,
    only_filters: bool = False,
    test: bool = False,
```